src/js/utils/StringDiff.js

```diff
@@ -0,0 +1,736 @@
+/* istanbul ignore next */
```

Comment:
@Bronson-Brown-deVost : I've learned about this `/* istanbul ignore next */` and have added it at a few places. This will allow us to ignore certain lines that we will not unit test.

For instance, this file is a string diff algorithm, not available on NPM. I think the Potrace is similar (although you've also edited that). I don't care to write unit tests against this but I also don't want it dragging down our code coverage.

I also ignored things like our AJAX helper since we won't test that functionality in a unit testing environment and route definitions. The routes we could perhaps test as part of integration testing, but we're not at that point.

Reply:
Cool!  We should be sparing with that, but yes it absolutely should go on your axios helper, and probably should go on the Potrace code as well (at least in the few section that I can't figure out how to get a test to pass over; essentially there are sections of that code, which may or may not be used based on the nature of the shape being traced).

Reply:
Exactly. You read my mind! It should only be used in those cases where we _won't_ be unit testing something ... not because we should but haven't :)